spec/requests/preferences/preference_spec.rb

```diff
@@ -81,4 +81,68 @@
       end
     end
   end
+
+  describe 'GET edit' do
+    context 'when logged in' do
+      let!(:user) { create(:user) }
+      let!(:preference) { create(:preference, user_id: user.id) }
+      let!(:id) do
+        preference.id
+      end
```

Comment:
I think you are not using this variable `id` defined here 🤔

Reply:
:astonished: 
Sorry about that, let me delete it in a new commit!